src/components/pages/ProfileList/profileList.scss

```diff
@@ -0,0 +1,25 @@
+.profile-list{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+
+
+    .profile-card{
```

Comment:
I like the styling you added for the profile card(s). It looked good when you showed it to us over zoom.